docs/modules/reference/pages/service-assurance/monitors/WmiMonitor.adoc

```diff
@@ -0,0 +1,132 @@
+
+| domain
+| Windows domain where the user is located.
+You do not have to use the domain parameter if you use local user accounts.
+| none
+
+3+|*Optional*
+
+| matchType
+| Tells the OpenNMS WMI system to deal with multiple instance results.
+Available match types are:
+
+* *all*: all instances must comply to the compare operation.
+* *none*: no instances should comply with the compare operation.
+* *some*: only some (one or more) must comply with the compare operation.
+* *one*: only one instance can comply with the compare operation.
+| all
+
+| wmiClass
+| Class of instances within WMI.
+This class contains objects representing properties and methods available.
+| Win32_ComputerSystem
+
+| wmiObject
+| Member of a WMI property set.
+For example, the `Win32_ComputerSystem` class contains properties that we can look at and poll.
+The default "WMI" service uses the "Status" property to determine if the system is running and if WMI is available.
+| Status
+
+| compareOp
+| The compare operation tells the OpenNMS WMI poller plugins how to verify the nature (up, critical) of a specific property within WMI.
+In the event of unequal comparisons, the system will always have the WMI value (as retrieved from the target) on the left.
+Available compare operations are:
+
+* *EQ* (equals)
+* *NEQ* (not equals)
+* *GT* (greater than)
+* *LT* (less than)
+* *NOOP* (no operation)
+
+//not sure what "manager" is referring to here.
+
+The NOOP operation is a special case in which the Manager always returns a result code of "OK."
+This is typically used in scenarios where the existence of a class or property is more interesting than the contents of the property.
+OpenNMS uses this in the collector to check and ensure that WMI is available before attempting to collect data.
+| EQ
+
+| compareValue
+| The value to use when performing a comparison against the wmiObject.
+| OK
+
+| service-name
+| Name of the service to monitor.
+ ##If there is a ~ at the beginning of the parameter, the rest of the string will be used as a regular expression pattern match, otherwise it will be considered an exact match.##
+| WMI
+
+| retry
+| Number of retries before the service is marked as down.
+| 0
+
+| timeout
+| Time in milliseconds to wait for a response.
+|
+
+| namespace
+| Every WMI class exists in a namespace.
+The default namespace for most common classes is root/cimv2.
+It is possible to refer to WMI objects from other namespaces such as root/MicrosoftActiveDirectory.
+| root/cimv2
+
+|===
+
+kbd:[{}] _indicates the parameter supports <<reference:service-assurance/introduction.adoc#ref-service-assurance-monitors-placeholder-substitution-parameters, placeholder substitution>>._
+
+This monitor implements the <<reference:service-assurance/introduction.adoc#ref-service-assurance-monitors-common-parameters, Common Configuration Parameters>>.
+
+== Example
+
+The following example shows how to enable monitoring that already discovered WMI services are still running.
```

Comment:
This is a valid example, though it seems like a circular check to use WMI to see if the WMI service is running.  If the service is down, the poller will fail due to not being able to read data, not that it read a value of down. :D

Reply:
LOL. Makes sense. I just copied it from somewhere else. Could you suggest a better example?